src/obsolete-reporter.js

```diff
@@ -0,0 +1,64 @@
+const fs = require("fs");
+const path = require("path");
+
+const TOUCHED_FILE_LIST_PATH = path.join(
```

Comment:
this file needs to be something that can be determined within... 

- `toMatchImageSnapshot`
- `onRunStart`
- `onRunComplete`

Ideally I think this would live in the cache directory but `onRunStart` and `toMatchImageSnapshot` don't have obvious ways to access that. `process.cwd()` is the only thing I could think of that doesn't require config gymnastics.